app/src/main/java/com/internship/move/model/OnBoardingRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.internship.move.model
+
+class OnBoardingRepository(private val onBoardingInternalStorageManager: OnBoardingInternalStorageManager) {
```

Comment:
You don't need a repository for every screen.
Usually, the repository is used for data.